guidelines/templates/resources/custom-resource.md

```diff
@@ -0,0 +1,37 @@
+---
+title: {CRD kind}
+type: Custom Resource
+---
+
+> This document is a ready-to-use template for a Custom Resource (CR) document that provides the sample Custom Resource and description of its elements for a given CRD. For details, see the existing CR documents for the [Installer](https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/040-installation-custom-resource.md) and the [Api-gateway](https://github.com/kyma-project/kyma/blob/master/docs/api-gateway/docs/011-api-custom-resource.md).
```

Comment:
This document is a ready-to-use template for a Custom Resource (CR) document that provides a sample Custom Resource and description of its elements. Additionally, the document points to the Custom Resource Definition (CRD) used to create CRs of the given kind. For reference, see the existing documents for the [Installation] CR and the [Api] CR.

Reply:
Applied with the plural (CRs) at the end, as we point to two custom resources.